docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
   leantime:
     image: leantime/leantime:latest
     container_name: leantime
+    restart: always
```

Comment:
I prefer ```restart: unless-stopped```

Reply:
Yes, this would be better, but we have to be consistent between web and db.

Reply:
then change the db one too